plugin/PDFView.qml

```diff
@@ -223,8 +311,7 @@ SilicaFlickable {
                 selection: pdfSelection
 
                 onLinkClicked: base.linkClicked(linkTarget, contextHook)
-                onGotoClicked: base.goToPage(page - 1, top, left,
-                                             Theme.paddingLarge, Theme.paddingLarge)
+                onGotoClicked: base.gotoClicked(page, top, left)
```

Comment:
Only handler for signal is in the same file. Could just have some internal function called here?